web/client/reducers/geostory.js

```diff
@@ -321,6 +323,12 @@ export default (state = INITIAL_STATE, action) => {
         const focusedContent = status ? {target, selector, hideContent, path} : undefined;
         return set(`focusedContent`, focusedContent, state);
     }
+    case CLOSE_TUTORIAL: {
+        return {
+            ...state,
+            isSettingsEnabled: false
+        };
+    }
```

Comment:
Why do you need to change the reducer of `geostory` and `mediaEditor` on `CLOSE_TUTORIAL` action?

Reply:
because i need to reset to the default/initial situation and some actions are bound with the reducer. like opening the settings panel or media editor